OpenSimRT/RealTime/include/experimental/ExternalAccelerationBasedPhaseDetector.h

```diff
@@ -33,7 +33,6 @@
 #include "GaitPhaseDetector.h"
```

Comment:
Sorry, I do not get it. Why do we have AccelerationBasedPhaseDetector and ExternalAccelerationBasedPhase detector?

Reply:
I will have to check the code, but do you think we can merge the two?

Reply:
The first estimates the foot acceleration from station points in the model, and the second uses actual  acceleration measurements from sensors. It could be merged, but I have to take a second look, because it might get more complicated that it needs to be.